templates/python/flake.nix

```diff
@@ -0,0 +1,45 @@
+{
+  inputs.capacitor.url = "git+ssh://git@github.com/flox/capacitor?ref=v0";
+  inputs.capacitor.inputs.root.follows = "/";
+  inputs.capacitor.inputs.nixpkgs.follows = "nixpkgs/nixpkgs-stable";
+
+  inputs.nixpkgs.url = "git+ssh://git@github.com/flox/nixpkgs-flox";
+  inputs.nixpkgs.inputs.capacitor.follows = "capacitor";
+
+  inputs.floxpkgs.url = "git+ssh://git@github.com/flox/floxpkgs";
+  inputs.floxpkgs.inputs.capacitor.follows = "capacitor";
+  inputs.floxpkgs.inputs.nixpkgs.follows = "nixpkgs";
+
+  nixConfig.bash-prompt = "[flox]\\e\[38;5;172mλ \\e\[m";
+
+  outputs = args @ {capacitor, ...}:
+    capacitor args (
+      {
+        inputs,
+        self,
+        ...
+      }: let
+        outputsWithoutFloxEnv = {
+          description = "Python template";
+          config = {
+            stabilities = {
```

Comment:
@ysndr is this equivalent to the nixpkgs plugin?

Reply:
If youre speaking of the nixpkgs plugin we have on the `plugin` branch of floxpkgs [1], then yes

[1]: https://github.com/flox/floxpkgs/blob/plugins/plugins/stabilities.nix

Reply:
Added that plugin instead, look good now?